HomeWork_4/sum_of_indexes.py

```diff
@@ -0,0 +1,19 @@
+# Задание №6
+# ✔ Функция получает на вход список чисел и два индекса.
+# ✔ Вернуть сумму чисел между между переданными индексами.
+# ✔ Если индекс выходит за пределы списка, сумма считается до конца и/или начала списка.
+
+
+def func_(data: list[int], i_1: int, i_2: int) -> int:
+    i_1 = i_1 if i_1 >= 0 else 0
+    i_2 = i_2 if i_2 <= len(data) else len(data)
+    res = 0
+    for i in data[i_1:i_2]:
+        res += i
+    return res
+
+
+my_list = [1, 2, 2, 4, 7, 9]
+print(f' Задан лист: {my_list = }')
+print(f' Сумма чисел между между индексами = {func_(my_list, 1, 8)} ')
```

Comment:
В условии не сказано, как нужно задавать индексы, то есть можно как угодно. Я задал так:
```suggestion
print(f' Сумма чисел между между индексами = {func_(my_list, 5, 1)} ')
```
И получил вот такой ответ. Почему?
![image](https://github.com/ElenaFat918/OnePythonGB/assets/49741194/a7f0ef3d-ff5c-486b-939b-ef96617408c1)